src/runtime/Makefile

```diff
@@ -553,14 +553,16 @@ define MAKE_KERNEL_VIRTIOFS_NAME
 $(if $(findstring uncompressed,$1),vmlinux-virtiofs.container,vmlinuz-virtiofs.container)
 endef
 
+GENERATED_CONFIG = $(abspath $(CLI_DIR)/config-generated.go)
```

Comment:
A nitpick here ... do those changes belong to this commit? Wouldn't be better to have those in a different one?

Reply:
Yes they are part of the commit since the v2 shim needs access to variables like `version` (which previously were only generated by the build for the runtime).

Reply:
Gotcha!

Reply:
I've added some explanation the commit.